AdtSampleApp/SampleClientApp/CommandLoop.cs

```diff
@@ -1043,33 +1042,37 @@ public async Task CommandLoadModels(string[] cmd)
         /// <param name='twin_id0'>
         /// Id of an existing twin
         /// </param>
-        public async Task CommandObserveProperties(string[] cmd)
```

Comment:
Method actually makes no async calls, creating a build warning. Resolved by removing `async` keyword and returning Task.CompletedTask instead.